login/visitwebpage.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2016 Canonical Ltd.
+// Licensed under the LGPLv3, see LICENCE file for details.
+
+package login
+
+import (
+	"errors"
+	"net/http"
+	"net/url"
+
+	"github.com/juju/cmd"
+	"github.com/juju/identity/idmclient"
+	"github.com/juju/usso"
+	"gopkg.in/macaroon-bakery.v1/httpbakery"
+)
+
+// VisitWebPage returns a function which will allow authentication with usso
+// via the cli.
+// The user will be prompted for username, password and any two factor authentication
+// code via the command line. Existing oauth tokens can be obtained, or new ones stored
+// using the given TokenStore. If no TokenStore is specified then the nopStore will be
```

Comment:
nopStore isn't exported, so this comment won't make sense to the reader of the godoc.

Perhaps something like this? (replacing text from "Existing auth tokens")

// If non-nil, the given TokenStore is used to persistently store the
// oauth token obtained during the login process so that less interaction
// may be required in future.